app/services/CampaignTransformer.scala

```diff
@@ -18,7 +18,7 @@ object CampaignTransformer {
     val endDate   = CapiSectionTransformer.deriveEndDateOrDefaultToNow(section)
 
     for {
-      sponsorName     <- CapiSectionTransformer.deriveSponsorName(section)
+      sponsorName     <- Some(CapiSectionTransformer.deriveCampaignName(section))
```

Comment:
Why not make it a `val` now?

Reply:
God knows - wasn't concentrating. Have changed.